apps/cowswap-frontend/src/modules/account/containers/Transaction/ActivityDetails.tsx

```diff
@@ -247,8 +258,21 @@ export function ActivityDetails(props: {
     outputToken = COW[chainId]
   }
 
+  // TODO: Get this from the API
+  const isCustomRecipient = true
```

Comment:
You can get this info from the `order` obj.
Haven't double checked the fields, but it should be `owner` and `receiver`

Reply:
Added your suggestion